INSTALL.md

```diff
@@ -159,4 +162,5 @@ source ~/.bashrc
    export PYTHONPATH=$PYTHONPATH:~/ros2_mara_ws/install/lib/python3/dist-packages
    export GAZEBO_MODEL_PATH=$GAZEBO_MODEL_PATH:~/ros2_mara_ws/src/MARA
    export GAZEBO_PLUGIN_PATH=$GAZEBO_PLUGIN_PATH:~/ros2_mara_ws/src/MARA/mara_gazebo_plugins/build/
+   export RMW_IMPLEMENTATION=rmw_opensplice_cpp
```

Comment:
If we add it to utils we do not need this line.

Reply:
We tried with ut_launch and did not work fine. Adding it to the provisioning script works and seems like a valid approach to me

Reply:
Does not work in utils. I would prefer to add this and state this clearly in the install and readme.